src/signing/petition_types/petition_factors_types/petition_factors_state.rs

```
@@ -147,7 +155,7 @@ mod tests {
             FactorSourceIDFromHash::fs0(),
         );
 
-        sut.did_skip(&factor_instance, false);
+        sut.skip(&factor_instance, false);
```

Add test for failure instead of skip